usaspending_api/api_contracts/contracts/v2/awards/accounts.md

```diff
@@ -11,24 +11,47 @@ This endpoint returns a list of federal accounts under a given award.
 
 ### List Federal Accounts [POST]
 + Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
     + Attributes
-        + `award_id`:`CONT_AWD_DEAC5206NA25396_8900_-NONE-_-NONE-` (required, string)
-            Award to return accounts for
-        + `page`: 1 (optional, number)
-            Page number to return
-        + `limit`: 10 (optional, number)
-            Maximum number to return
-        + `order`: `desc` (optional, string)
-            Direction of sort
-        + `sort`:`total_transaction_obligated_amount` (optional, string)
-            The field to sort on
+        + `award_id` (required, string) - Award to return accounts for
+        + `page` (optional, number) - Page number to return
+        + `limit` (optional, number) - Maximum number to return
+        + `order` (optional, enum[string]) - Direction of sort
+            + Members
```

Comment:
Im guessing members is just options?

Reply:
Yeah, it is more descriptive to use enum[string] instead of an open string when there are limited accepted values.